### ISSUE TITLE
Any recommended background reading for panmorphic types

### ISSUE THREAD

**Issue by novocodev** [sparist/Om#26]
Are there any references/papers for reading more about "panmorphic types", google didn't return anything useful.

**Comment by sparist** [comment]
Nope; I invented the term. The best description I can give is the one in the documentation here:

http://www.om-language.org/index.html#language__data__

But I'm happy to answer any questions.
